tests/integration/PayoutCreateTest.php

```diff
@@ -58,6 +61,9 @@
             ],
             'date_of_birth' => '1990-01-31',
         ],
+        'scheme_selection' => [
+            'type' => 'instant_only',
+        ],
```

Comment:
Could we use a separate test with a dataset like the `sends user selected scheme selection` test? to test all the scheme selection types.